scripts/globals/mobskills/Tribulation.lua

```diff
@@ -0,0 +1,29 @@
+---------------------------------------------
+--  Tribulation
+--
+--  Description: Inflicts Bio and blinds all targets in an area of effect.
+--  Type: Enfeebling
+--  Utsusemi/Blink absorb: Ignores shadows
+--  Range: AoE
+--  Notes: Bio effect can take away up to 39/tick.
+---------------------------------------------
+
+require("scripts/globals/settings");
+require("scripts/globals/status");
+require("scripts/globals/monstertpmoves");
+
+---------------------------------------------
+
+function onMobSkillCheck(target,mob,skill)
+    return 0;
+end;
+
+function onMobWeaponSkill(target, mob, skill)
+
+    local typeEffect = EFFECT_BLINDNESS;
+
+    skill:setMsg(MobStatusEffectMove(mob, target, typeEffect, 20, 0, 120));
+    MobStatusEffectMove(mob, target, EFFECT_BIO, 39, 0, 120);

```

Comment:
i summon @teschnei or @maxtherabbit (should probably push a separate message for the other effect resist/landing)